src/drag-drop-polyfill.ts

```diff
@@ -18,1636 +20,1640 @@ type WhatWGAddEventListener = (
-                    }
-                    // Otherwise, run the appropriate step from the following list:
-                    else {
-
-                        // NO DROPZONE SUPPORT SINCE NATIVE IMPLEMENTATIONS IN BROWSERS ALSO DO NOT
-                        //console.log( "dnd-poly: dragenter not prevented, searching for dropzone.." );
-                        //var newTarget = DragOperationController.FindDropzoneElement( this.immediateUserSelection );
-
-                        // THIS IS SKIPPED SINCE SUPPORT IS ONLY AVAILABLE FOR DOM ELEMENTS
-                        // If the current target element is a text field (e.g. textarea, or an input element whose type attribute is in the Text state) or an
-                        // editable element, and the drag data store item list has an item with the drag data item type string "text/plain" and the drag data
-                        // item kind Plain Unicode string
-                        //if( ElementIsTextDropzone( this.immediateUserSelection, this.dragDataStore ) ) {
-                        //Set the current target element to the immediate user selection anyway.
-                        //this.currentDropTarget = this.immediateUserSelection;
-                        //}
-                        //else
-                        // If the current target element is an element with a dropzone attribute that matches the drag data store
-                        //if( newTarget === this.immediateUserSelection &&
-                        //    DragOperationController.GetOperationForMatchingDropzone( this.immediateUserSelection, this.dragDataStore ) !== "none" ) {
-                        // Set the current target element to the immediate user selection anyway.
-                        //    this.currentDropTarget = this.immediateUserSelection;
-                        //}
-                        // If the immediate user selection is an element that itself has an ancestor element
-                        // with a dropzone attribute that matches the drag data store
-                        // NO DROPZONE SUPPORT SINCE NATIVE IMPLEMENTATIONS IN BROWSERS ALSO DO NOT
-                        //else if( newTarget !== null && DragOperationController.GetOperationForMatchingDropzone( newTarget, this.dragDataStore ) ) {
-
-                        // If the immediate user selection is new target, then leave the current target element unchanged.
-
-                        // Otherwise, fire a DND event named dragenter at new target, with the current target element
-                        // as the specific related target. Then, set the current target element to new target,
-                        // regardless of whether that event was canceled or not.
-                        //this.dragenter( newTarget, this.currentDropTarget );
-                        //this.currentDropTarget = newTarget;
-                        
```

Comment:
the definition of this export wrapper will result in global usage pattern of
```TS
/// <reference path="../../src/drag-drop-polyfill.d.ts" />

DragDropPolyfill.DragDropPolyfill.Initialize();
```

Rather just export the Initialize() function to get global usage like
```TS
/// <reference path="../../src/drag-drop-polyfill.d.ts" />

DragDropPolyfill.Initialize();
```

Module usage
```TS
import {Initialize} from "drag-drop-polyfill";

Initialize();
```
or 
```TS
import {Initialize as DragDropPolyfill} from "drag-drop-polyfill";

DragDropPolyfill();
```